keyboards/karn/readme.md

```diff
@@ -0,0 +1,42 @@
+# Karn 2
+
+Split, ortholinear, non-staggered, diodeless, 38 key keyboard, with a unique set of 1.5u thumb arc keys and 1.5u pinky reach keys. 
+
+* Keyboard Maintainer: [Robert Mills](https://github.com/robcmills)
+* Hardware Supported: Blackpill STM32F401
+
+Based on the excellent [cantor](https://github.com/diepala/cantor), and inspired by the popular [corne](https://github.com/foostan/crkbd), [ferris](https://github.com/pierrechevalier83/ferris) and [sweep](https://github.com/davidphilipbarr/Sweep) keyboards.
+
+[https://github.com/robcmills/karn-keyboard](https://github.com/robcmills/karn-keyboard)
+
+![karn-2-left](https://i.imgur.com/vm6XFyI.jpeg)
+
+![karn-2-full](https://i.imgur.com/R7WnlC3.jpeg)
+
+![karn-2-pcb](https://i.imgur.com/EQqyyED.png)
```

Comment:
This uses a smaller, but high res version of the images, that is more suited to github's markdown
```suggestion
![karn-2-left](https://i.imgur.com/vm6XFyIh.jpeg)

![karn-2-full](https://i.imgur.com/R7WnlC3h.jpeg)

![karn-2-pcb](https://i.imgur.com/EQqyyEDh.png)
```